src/main/java/com/marklogic/spark/writer/MarkLogicWrite.java

```diff
@@ -66,17 +63,15 @@ public StreamingDataWriterFactory createStreamingWriterFactory(PhysicalWriteInfo
 
     @Override
     public void commit(long epochId, WriterCommitMessage[] messages) {
-        logger.info("EpochId for commit is "+ epochId);
         if (messages != null && messages.length > 0) {
-            logger.info("Commit messages received: {}", Arrays.asList(messages));
+            logger.info("Commit messages received for epochId {}: {}", epochId, Arrays.asList(messages));
```

Comment:
Combined this into one log message; one log event is typically preferable for tools like Grafana and other log analyzers.

Reply:
We might need to log epochId, even when the commit messages are null or empty.

Reply:
I changed it to always log epochId and messages. I'm going to do some testing to see when messages are null, as I've encountered that before. Not sure if it matters to log messages when it's null, that's probably fine.